lnwallet/chancloser/rbf_coop_transitions.go

```diff
@@ -584,6 +584,12 @@ func (c *ClosingNegotiation) ProcessEvent(event ProtocolEvent, env *Environment,
 	// we receive a confirmation event, or we receive a signal to restart
 	// the co-op close process.
 	switch msg := event.(type) {
+	// Ignore any potential duplicate channel flushed events.
+	case *ChannelFlushed:
+		return &CloseStateTransition{
+			NextState: c,
+		}, nil
```

Comment:
Why are we getting duplicate flushed events?? This makes me concerned about other parts of the design.